### PR TITLE
Add support for representer

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: lib/tooling_invoker/representer_job.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    def initialize(id, *args)
+      super(id, *args)
+    end
+
+    def invocation_args
+      ["bin/generate.sh", exercise, "/mnt/exercism-iteration/", "/mnt/exercism-iteration/"]
+    end
+
+    def results_filepath
+      "results.json"
```
[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply lib/tooling_invoker/representer_job.rb:12]: Good catch.

File: lib/tooling_invoker/representer_job.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    def working_directory
+      "/opt/representer"
+    end
+
+    def tool
+      "#{language}-representer"
+    end
+
+    def parsed_result
```
[the remark this answers is not in the document]
Comment (iHiD) [reply lib/tooling_invoker/representer_job.rb:23]: I think this method can just return `result` for now?

[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply lib/tooling_invoker/representer_job.rb:23]: Agreed.

**Comment by iHiD** [comment]
This is still draft FYI. (You probably know that)

File: test/worker_test.rb [hunk 3]
```diff
@@ -2,29 +2,103 @@
...
-      job_id = "123"
-      language = "ruby"
-      exercise = "bob"
-      s3_uri = "s3://..."
-      container_version = "v1"
-      execution_timeout = 10
+    def setup
+      super
 
+      @job_id = "123"
```
Comment (ErikSchierboom) [inline test/worker_test.rb:8]: I've added tests to verify the representer and analyzer flows. I've extracted some of the common code to the `setup` method. Let me know if that is the right thing to do.

**Review by ErikSchierboom — COMMENTED** [review]
I've added some tests

File: test/worker_test.rb (hunk 3 above)
Comment (iHiD) [reply test/worker_test.rb:8]: Yeah - that makes sense 🙂

**Comment by iHiD** [comment]
Merged. Nice work! :)

**Pull request merged**